packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -351,21 +348,25 @@ export class ClusterReachability extends EventsScope {
 
       if (e.candidate) {
         if (e.candidate.type === CANDIDATE_TYPES.SERVER_REFLEXIVE) {
-          this.saveResult('udp', latencyInMilliseconds, e.candidate.address);
+          let serverIp = null;
+          if ('url' in e.candidate) {
+            const regex = /stun:([\d.]+):\d+/;
+
+            const match = (e.candidate as any).url.match(regex);
+            if (match) {
+              // eslint-disable-next-line prefer-destructuring
+              serverIp = match[1];
+            }
+          }
+
+          this.saveResult('udp', latencyInMilliseconds, e.candidate.address, serverIp);
```

Comment:
Suggestion: use ternary operator

```suggestion
const match = (e.candidate as any).url.match(regex);
match && ([, serverIp] = match);

this.saveResult('udp', latencyInMilliseconds, e.candidate.address, serverIp);
```

Reply:
I'm not sure if this line is good readable.. `match && ([, serverIp] = match);`